### ISSUE TITLE
Add source file to informational hover

### ISSUE THREAD

**Issue by cemerick** [ocamllabs/vscode-ocaml-platform#378]
Just switched over from reason-vscode, and I _think_ this is one of just 2-3 items that I miss.

Currently, the informational hover shows a symbol's type and any associated documentation. It would be great if the location of the type's definition were appended. This helps to quickly determine which module something is from when there are many same-named terms available from various modules, maybe from overlapping `open`s, etc.

(including reason-vscode's main screenshot, which conveniently shows the source path in its hover)

![image](https://user-images.githubusercontent.com/47489/94445831-e9c90980-0175-11eb-897a-1a8b04bb5d28.png)

**Comment by mnxn** [comment]
I believe this is something that would have to be implemented on the [language server side](https://github.com/ocaml/ocaml-lsp/).

**Comment by rgrinberg** [comment]
Do we know which field of the payload provides this information? Happy to take a look on the LSP side.

**Comment by mnxn** [comment]
What payload are you referring to? Shouldn't it be possible to use the same mechanism as goto definition to get the filename and insert it into the hover information?

**Comment by rgrinberg** [comment]
Indeed. I thought there was a way to perhaps it into a link that the user click. But it appears to be just text in this screenshot
